haskell-foundation.cabal

```diff
@@ -1,14 +1,14 @@
 name:               haskell-foundation
 version:            0.1.0.0
 build-type:         Simple
-cabal-version:      >= 1.10
+cabal-version:      2.0
 
 executable site
   main-is:          site.hs
   build-depends:    base == 4.*
-                  , hakyll
+                  , hakyll ^>=4.15
```

Comment:
The `cabal` solver was picking absurd solutions on my machine that involved ancient Hakyll versions and very new Pandoc versions and that would not build together. This seems to fix it.